test-network-nano-bash/README.md

```diff
@@ -0,0 +1,83 @@
+# Test network - Nano bash
+
+Test network Nano bash provides a set of minimal bash scripts to run a Fabric network on your local machine.
+The network is functionally equivalent to the docker-based Test Network, you can therefore run all the tutorials and samples that target the Test Network with minimal changes.
+The Fabric release binaries are utilized rather than using docker containers to avoid all unnecessary layers. Only the chaincode and chaincode builder runs in a docker container behind the scenes.
+Using the Fabric binaries also makes it simple for Fabric developers to iteratively and quickly modify Fabric code and test a Fabric network as a user.
+
+As the name `nano` implies, the scripts provide the smallest minimal setup possible for a Fabric network while still offering a multi-node TLS-enabled network:
+- Minimal set of dependencies
+- Minimal requirements on Fabric version (any v2.x orderer and peer nodes are supported)
+- Minimal set of environment variable overrides of the default orderer orderer.yaml and peer core.yaml configurations
+- Minimal scripting with minimal set of reference commands to get a Fabric network up and running
+- Minimal channel configuration for an orderer organization (3 ordering nodes) and two peer organizations (with two peers each)
+- Minimal endorsement policy to allow a single organization to approve and commit a chaincode (unlike Test Network which requires both organizations to endorse)
+
+# Prereqs
+
```

Comment:
I had to run `make clean` in my existing Fabric environment before I could run this. Not sure how many others will hit that problem, but it may be worth calling out.

Reply:
@nikhil550 Did you have the binaries in the fabric-samples/bin directory? The PATH should pick them up there if you don't have binaries built in fabric/build/bin, that would be the 'normal' experience for consumers rather than Fabric developers.

Reply:
I did, but the sample looked for the binaries in the Fabric folder first.